properties/src/main/java/com/fasterxml/jackson/dataformat/javaprop/JavaPropsSchema.java

```diff
@@ -396,9 +423,10 @@ public String pathSeparator() {
         return _pathSeparator;
     }
 
-    /**
-     * @since 2.10
-     */
+    public char pathSeparatorEscapeChar() {
```

Comment:
Let's leave `@since 2.10` next to `prefix()`, add one for 2.14 above new method.

Reply:
Whoops.